packages/apps/desktop-viewer-test/src/frontend/components/modelSelector/SelectIModel.tsx

```diff
@@ -199,9 +200,10 @@ export const SelectIModel = ({
 
   return (
     <div className="itv-scrolling-container select-imodel">
-      <div className={"itv-content-margins"}>
+      <PageLayout.TitleArea className={"itv-content-margins"}>
```

Comment:
so is this PageLayout.TitleArea inside a PageLayout.Content ?
does that look right? should it be a parent/child or siblings? 

maybe it looks fine. i havent pulled, but just wondering based off your code

also, i think might be nice to add a title area for the select project pg to say something about what itwin/projects imodels are being shown